base/linalg/lapack.jl

```diff
@@ -721,7 +733,7 @@ for (gesvx, elty, relty) in
             work      = Array($elty, 4n)
```

Comment:
As I read it both NETLIB http://www.netlib.org/lapack/explore-html/df/dcf/zgesvx_8f.html#abe3ae17895d46aeb7c9e5a076eaeeb69, and Intel say `2n` in the complex case.